GoogleChromeLabs/ps-analysis-tool
# Add UTM params to outgoing links

## Description

Add UTM params to outgoing links.

## Relevant Technical Choices

- Create a new utility function `addUTMParams` in common package, so that it can be used in both CLI and extension.
- Use the function wherever the link needs to be updated.

## Testing Instructions

- Go through all pages of the extension and check all outgoing links ( https://developer.chrome.com/, https://privacysandbox.com/, https://youtube.com ) if they have utm parameters attached to them.
- No utm params were added to other links like github.com etc
- Ensure that no links are broken due to this change.

## Additional Information:



## Screenshot/Screencast

<img width="651" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/cde20d0a-edcd-41f3-963f-ff01d87fb7d8">

---

## Checklist






Fixes #506
